tests/pyoozie/test_builder.py

```diff
@@ -230,5 +229,6 @@ def test_coordinator_builder(coordinator_xml_with_controls, workflow_app_path):
         })
 
     # Can it XML?
-    expected_xml = builder.build()
-    assert xml_to_dict_unordered(coordinator_xml_with_controls) == xml_to_dict_unordered(expected_xml)
+    expected_xml = coord_builder.build()
+    assert tests.utils.xml_to_dict_unordered(coordinator_xml_with_controls) == \
+        tests.utils.xml_to_dict_unordered(expected_xml)
```

Comment:
This would be more readable if you assigned the LHS and RHS to locals and then asserted equality. (You could also make it marginally better by using `from test import utils`, but I wouldn't worry about that.)